conf/build.manifest

```diff
@@ -43,6 +43,7 @@ apt-repos:
 # NOTE: Installed in the order listed
 ############################################################################
 base-packages:
+- linux-headers-truenas-amd64
```

Comment:
this is almost sorted alphabetically, lets keep the trend unless its required to be first